### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [micwallace/reddinator#30]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Reddinator](https://play.google.com/store/apps/details?id=au.com.wallaceit.reddinator&hl=it)
3. Activate the registration from the Accessibility Scanner and use the Reddinator app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201119_145903](https://user-images.githubusercontent.com/25429624/99677243-ca65a300-2a79-11eb-952f-8fcedd3b3e83.png)
